docs/usage/key.rst

```diff
@@ -26,6 +26,11 @@ Examples
     Remember your passphrase. Your data will be inaccessible without it.
     Key updated
 
+    # Import a previously-exported key into the specified
+    # key file (creating or overwriting the output key)
```

Comment:
maybe add a dot at the end of the sentence.

Reply:
I'm following the convention of other comments in this documentation (e.g. lines 9 and 21) which do not have dots ending the sentences.

Reply:
hmm, ok, maybe this should be cleaned up later and in general.

i try to follow the rule "if it is a complete sentence, then it should have a full-stop at the end".